ci-operator/step-registry/ipi/conf/aws/proxy/ipi-conf-aws-proxy-commands.sh

```diff
@@ -260,8 +246,11 @@ PROXY_IMAGE=registry.svc.ci.openshift.org/origin/4.5:egress-http-proxy
 
 PROXY_NAME="$(/tmp/yq r "${CONFIG}" 'metadata.name')"
 REGION="$(/tmp/yq r "${CONFIG}" 'platform.aws.region')"
+echo Using region: ${REGION}
+test -n "${REGION}"
 
-RHCOS_AMI="$(jq -r ".amis[\"${REGION}\"].hvm" /var/lib/openshift-install/rhcos.json)"
+curl -L -o /tmp/fcos-stable.json https://builds.coreos.fedoraproject.org/streams/stable.json
+AMI=$(jq -r .architectures.x86_64.images.aws.regions[\"${REGION}\"].image < /tmp/fcos-stable.json)
```

Comment:
Can we follow this up with a `test -z "${AMI}"` guard to error out if there is no FCOS AMI in the target region?

Reply:
Resolved in f4541a0 -> 634798fce4ab.